source/jwtd/jwt.d

```diff
@@ -106,6 +109,8 @@ JSONValue decode(string token, string key) {
 	return payload;
 }
 
+version (UseBotan) { }
+else {
```

Comment:
The block below should be indented, but it will ruin the review so I let it be as it was